aml-development/frontend
# feat(profile): allow users to view other user's profiles

AMLOS-753

**Description**
As a user I want to view another user's profile to view their contact info, org, avatar, and possibly applications they own.

**Acceptance Criteria**
A user can go to a link (through a listing, or directly) that will display the profile info of another user

**How To Test**
Pull this branch
Pull `add_avatar_fields` from `ozp-backend` if https://github.com/aml-development/ozp-backend/pull/444 is not merged

On a listing page, you should see links under _Owners_, _Reviews_, and _Listing Administration --> Listing Changes_ to user profiles.  Clicking a link should direct you to the user's profile page.

All profile links should show the user's avatar icon, or the default user icon if the user does not have an avatar.

Changing the URL manually should load the correct pages, ie:
`http://localhost:8080/#/profile/13` --> jones's profile page
`http://localhost:8080/#/profile` --> logged in user's profile page
`http://localhost:8080/#/profile/edit` --> logged in user's edit profile page
